src/NuGet/BuildNuGets.csx

```diff
@@ -98,6 +99,7 @@ string[] RedistPackageNames = {
     "Microsoft.CodeAnalysis.Workspaces.Common",
     "Microsoft.VisualStudio.LanguageServices",
     "Microsoft.VisualStudio.LanguageServices.Next",
+    "Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient",
```

Comment:
The comment at L116 explains that these need to be listed in both `RedistPackageNames` and `PreReleaseOnlyPackages`